grammars/javascript.cson

```diff
@@ -1096,7 +1080,11 @@
           '1':
             'name': 'meta.delimiter.method.period.js'
           '2':
-            'name': 'entity.name.function.js'
+            'patterns': [
```

Comment:
Unfortunately there's no docs.  Most of it is like you said directly from Textmate, and the majority of Atom's builtin syntaxes were converted from the Textmate ones.